Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -1149,32 +1148,36 @@ class AtlasStepper {
   /// Perform the actual step on the state
   ///
   /// @param state is the provided stepper state (caller keeps thread locality)
-  template <typename propagator_state_t, typename navigator_t>
-  Result<double> step(propagator_state_t& state,
-                      const navigator_t& /*navigator*/) const {
+  /// @param propDir is the direction of propagation
+  /// @param material is the material properties
+  /// @return the result of the step
+  Result<double> step(State& state, Direction propDir,
+                      const IVolumeMaterial* material) const {
```

Comment:
Is there a good place do document that this is allowed to be `nullptr`?

Reply:
hm I think this is a common problem with our templated interfaces. I guess I can only copy it to all instances